src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java

```diff
@@ -50,30 +50,84 @@
  * </li>
  * </ul>
  * <p>
- * To prevent {@code FooCheck} violations from being reported write:
+ * To use default module configuration:
  * </p>
  * <pre>
- * &#64;SuppressWarnings("foo") interface I { }
- * &#64;SuppressWarnings("foo") enum E { }
- * &#64;SuppressWarnings("foo") InputSuppressWarningsFilter() { }
+ * &lt;module name=&quot;TreeWalker&quot;&gt;
+ *   &lt;module name=&quot;MemberName&quot;/&gt;
+ *   &lt;module name=&quot;ConstantName&quot;/&gt;
+ *   &lt;module name=&quot;ParameterNumber&quot;/&gt;
+ *   &lt;module name=&quot;NoWhitespaceAfter&quot;/&gt;
+ *   &lt;module name=&quot;SuppressWarningsHolder&quot;/&gt;
+ * &lt;/module&gt;
+ * &lt;module name=&quot;SuppressWarningsFilter&quot;/&gt;
+ * </pre>
+ * <pre>
+ * Class Test {
```

Comment:
`class`

java code should be compilable.

Reply:
done